### ISSUE TITLE
Doesn't support get helper with input element

### ISSUE THREAD

**Issue by BryanCrotaz** [ember-template-lint/ember-template-lint#1283]
Using get helper to read the value of an input element is not supported
`(get _ "target.value")`

no-implicit-this rule triggers:
`Ambiguous path '_' is not allowed.`

**Comment by rwjblue** [comment]
Does not seem like a bug in ember-template-lint to me (or you haven't given enough context for me to understand :wink:). What is `_` here? Are you referring to the built-in `get` helper from Ember itself? What do you expect this invocation to do?

**Comment by BryanCrotaz** [comment]
As I understand it, the underscore represents `this`?

example from `ember-set-helper`:
```
<button
  value="Hola!"
  {{on "click" (set this.greeting (get _ "target.value"))}}
>
  Español
</button>
```

**Comment by rwjblue** [comment]
There is no special behavior RE: `_` with Ember's `get` helper. The special placeholder syntax that you see in the `ember-set-helper` README is provided by `ember-set-helper` internally, and is "magically" (from the context of the template context itself) creating a new variable called `_` via AST transform. There is no way that ember-template-lint can know about this (to avoid the no-implicit-this warning), though I think you should be able to configure the rule so that it assumes `_` is a helper (and therefore will not warn about it's usage).

@pzuraq - You might want to clarify in that addon's README about a conflict with the default octane linting configuration when using the placeholder syntax.
